### DISCUSSION TITLE
Love this.

### DISCUSSION THREAD

**Discussion by chadbr** [project-nv/night-vision#55] [General]
Awesome project - v2's are always the best.

**Comment by jacekjursza** [comment]
This is simply a masterpiece. Thank you for this code. Even such an early version as 0.3.2 this is everything and even more than I needed ;) The fact that the chart is so 'hackable' means I can get everything I need, which is testament to the extraordinary quality of the code.

**Comment by jacekjursza** [comment]
PS. Taking look at the roeadmap - IMO it would be better to focus on all other things except "Create a built-in indicator collection", as these can be calculated and probably will be calculated anyway by the external part of the app (in order to use it as part of the trading algos).

**Reply by oneart-dev** [reply]
i strongly disagree here.
